providers/flagd/src/test/java/dev/openfeature/contrib/providers/flagd/FlagdProviderTest.java

```diff
@@ -111,14 +106,15 @@ void path_arg_should_build_domain_socket_with_correct_path() {
                         (mock, context) -> {
                         })) {
                     when(NettyChannelBuilder.forAddress(any(DomainSocketAddress.class))).thenReturn(mockChannelBuilder);
-                    new FlagdProvider(path);
+
+                    new FlagdProvider(FlagdOptions.builder().socketPath(path).build());
 
                     // verify path matches
                     mockStaticChannelBuilder.verify(() -> NettyChannelBuilder
                             .forAddress(argThat((DomainSocketAddress d) -> {
                                 assertEquals(d.path(), path); // path should match
                                 return true;
-                            })), times(2));
+                            })), times(1));
```

Comment:
Note - we now use a shared channel [1]  which is the recommendation [2]

> Always re-use stubs and channels when possible.

[1] - https://github.com/open-feature/java-sdk-contrib/blob/main/providers/flagd/src/main/java/dev/openfeature/contrib/providers/flagd/FlagdProvider.java#L105
[2] - https://grpc.io/docs/guides/performance/